### PR TITLE
Quiz form styling

### PR DESCRIPTION
**Pull request by ndittren**
* Fix alignment issue with the form
* Added google font
* Make the date input smaller
* Disable/enable calendar script
* Restrict student access to the dashboard and quiz pages

### CODE REVIEW

File: quizcon/templates/main/courses.html [hunk 1]
```diff
@@ -6,45 +6,57 @@
...
-                    {% endif %}</td>
-                    <td>{% if course.info.termyear %}{{ course.info.termyear }}{% else %} &ndash; {% endif %}</td>
-                    <td>{{ course|course_faculty_string }}</td>
-                </tr>
-                {% endfor %}
-            </tbody>
-        </table>
-    </div>
-</div>
+    {% if is_faculty %}
```
Comment (sdreher) [inline quizcon/templates/main/courses.html:9]: `is_faculty` is not in the `DashboardView` context. This view will always render as "forbidden"

File: quizcon/templates/courseaffils/course_detail_student.html [hunk 2]
```diff
@@ -1,6 +1,10 @@
-<div class="row pt-3 px-0 mt-0 mx-0">
-    <div class="col-12">
-        {{course.title}}
-        <div><b>Your role:</b> {% if is_faculty %}Instructor{% else %}Student{% endif %}</div>
+<div class="row justify-content-md-center">
+    <div class="col-md-auto text-center my-5">
+        <h2>403: Forbidden</h2>
+
+        <p>
+          We're sorry. This content is not available for students.
```
Comment (sdreher) [inline quizcon/templates/courseaffils/course_detail_student.html:6]: A user who is not a class member will also hit this error message. Do we want to broaden the message?

**Review by sdreher — COMMENTED** [review]
A few permission-related nits to address.

File: quizcon/templates/main/courses.html (hunk 1 above)
Comment (ndittren) [reply quizcon/templates/main/courses.html:9]: @sdreher Thinking this through, I realized that someone can be both a student and faculty. Having this restriction doesn't necessarily make sense. I've reverted it back to just showing a person's courses.

**Pull request merged**